README.md

```diff
@@ -288,6 +290,11 @@ Executing the tests:
 Changelog
 =========
 
+dev
+-----
+
+- `load_dotenv` and `dotenv_values` now accepts `encoding` paramater (#161)
```

Comment:
Could use `locale.getpreferredencoding()` as default value. This will keep behavior stay same as before, because it's default encoding when we call `open`.

Reply:
You can simply pass `encoding=None` to let `io.open` infer the user’s default environment encoding.

Reply:
Passing encoding=None sounds like the way to go, breaking the existing API doesn't sound like a good idea.

Reply:
I've updated the encoding to be platform independent now, it's not a breaking change now. cc: @bbc2